XmlSlurper is too strict when reading POM files for license report

Gradle: 7.0.2
License report plugin: 1.74
Problematic dependency: istack-commons-3.0.11.pom

This happens out of the box, no configuration.

```
Caused by: org.gradle.api.GradleException: The prefix "Xlint" for element "Xlint:all" is not bound.
	at com.github.vlsi.gradle.license.GatherLicenseTask.findPomLicenses(GatherLicenseTask.kt:546)
	at com.github.vlsi.gradle.license.GatherLicenseTask.run(GatherLicenseTask.kt:332)
...
	Suppressed: org.xml.sax.SAXParseException; systemId: file://<redacted>/.gradle/caches/modules-2/files-2.1/com.sun.istack/istack-commons/3.0.11/b1b7e589e116b7db7217cc2098223185690e740b/istack-commons-3.0.11.pom; lineNumber: 408; columnNumber: 38; The prefix "Xlint" for element "Xlint:all" is not bound.
...
at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1251)
at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)
at groovy.util.XmlSlurper.parse(XmlSlurper.java:203)
at groovy.util.XmlSlurper.parse(XmlSlurper.java:221)
at com.github.vlsi.gradle.license.PomLicenseLoaderKt.parseXml(PomLicenseLoader.kt:111)
at com.github.vlsi.gradle.license.PomLicenseLoaderKt.access$parseXml(PomLicenseLoader.kt:1)
at com.github.vlsi.gradle.license.PomLicenseLoaderKt$loadLicenses$1$1$1$1.invokeSuspend(PomLicenseLoader.kt:139)
...
```

--- a/README.md
+++ b/README.md
@@ -191,6 +191,7 @@ Change log
 ----------
 v1.78
 * chore: bump Gradle 6.7 -> 6.9.1
+* license-gather: ignore xml namespaces when parsing POM files (see #43)
 
 v1.77
 * crlf-plugin: bump jgit to 5.13.0.202109080827-r

--- a/plugins/license-gather-plugin/src/main/kotlin/com/github/vlsi/gradle/license/PomLicenseLoader.kt
+++ b/plugins/license-gather-plugin/src/main/kotlin/com/github/vlsi/gradle/license/PomLicenseLoader.kt
@@ -108,7 +108,7 @@ fun GPathResult.attr(name: String): String = get("@$name").text()
 @Suppress("UNCHECKED_CAST")
 fun GPathResult.getList(name: String) = getProperty(name) as Iterable<GPathResult>
 
-private fun File.parseXml(): GPathResult = XmlSlurper().parse(this)
+private fun File.parseXml(): GPathResult = XmlSlurper(false, false).parse(this)
 
 fun GPathResult.parsePom(): PomContents {
     fun GPathResult.parseId(parentGroup: String = "") =

